<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>desWidgetClass</class>
 <widget class="QMainWindow" name="desWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>desWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="boutonLancerDe">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tirer le dé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="afficheDe">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher la valeur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remiseZero">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rémise à zéro du score</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scoreTot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Score total</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultat">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="afficheScore">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufichier">
    <property name="title">
     <string>fichier</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuchoisir">
    <property name="title">
     <string>choisir</string>
    </property>
    <addaction name="actiondes"/>
    <addaction name="actiondehisto"/>
   </widget>
   <addaction name="menufichier"/>
   <addaction name="menuchoisir"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiondes">
   <property name="text">
    <string>des</string>
   </property>
  </action>
  <action name="actiondehisto">
   <property name="text">
    <string>dehisto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="desWidget.qrc"/>
 </resources>
 <connections/>
</ui>
